packages/angular-table/src/constructTableHelper.ts

```diff
@@ -14,13 +14,13 @@ import type { Signal } from '@angular/core'
  */
 export type TableHelperOptions<
   TFeatures extends TableFeatures,
-  TData extends RowData = any,
+  TDataList extends Array<RowData> = Array<any>,
```

Comment:
just checking, it shouldn't be `= Array<RowData>`? instead of `any`?

Reply:
I was carrying over the `= any` from before for now. I'm not yet sure what is correct here though.